Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

```
@@ -0,0 +1,180 @@
+  name: Medical Prescription
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    content: |2
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Medical Prescription[/head]
+
+      [head=3]Patient Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Position Title:[/bold] 
+          [bold]Vessel IFF:[/bold] 
+
+      [head=3]Prescription[/head]
+          [bullet/] CHEMICAL, dosage: XX u
+          [bullet/] CHEMICAL, dosage: XX u
+
+          [bold]WRITE ANY INSTRUCTIONS HERE.[/bold]
+
+
+
+      [color=#AAAAAA]This Form is void without the signature and stamp of a medical specialist[/color]
+
+- type: entity # Based on form made by Radiantflash (Lykaon Rahmer)
+  id: PaperWrittenMedicalFormHealthInsurance
+  name: Health Insurance Contract
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    content: |2
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Health Insurance Contract[/head]
+
+      [head=3]Policy Holder Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Occupation:[/bold] 
+          [bold]Vessel IFF:[/bold] 
+
+      [head=3]Plan Coverage[/head]
+          [bold]Medical Coverage:[/bold]
+          \[  ] Treatment: Coverage up to $5 000. ($2 000 premium)
+          \[  ] Revival: Coverage up to $10 000. ($2 000 premium)
+          \[  ] Cloning Fee: Coverage up to $25 000. ($5 000 premium)
+
+          [bold]Additional Coverage:[/bold]
+          \[  ] Towing: Coverage up to $10 000. Small shuttles only. ($2 000 premium)
+
+      [head=3]Grand Total[/head]
+          [bold]$XXX XXX[/bold]
+
+
+      [color=#AAAAAA]This Form is void without the signatures of borth Parties and the stamp of a medical specialist[/color]
+
+- type: entity
+  id: PaperWrittenMedicalFormServiceInvoice
+  name: Medical Service Invoice
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    content: |2
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Medical Service Invoice[/head]
+
+      [head=3]Patient Information[/head]
+            [bold]Name:[/bold] 
+            [bold]Species:[/bold] 
+            [bold]Gender:[/bold] 
+            [bold]Occupation:[/bold] 
+
+      [head=3]Responder Information[/head]
+            [bold]Name:[/bold] 
+            [bold]Vessel IFF:[/bold] 
+            [bold]Shift Time:[/bold] 
+
+      [head=3]Invoice[/head]
+      - $XX XXX - Treatment
+      - $XX XXX - Revival
+      - $XX XXX - Cloning
+      - $XX XXX - Towing
```

```suggestion
      $XX XXX - Treatment
      $XX XXX - Revival
      $XX XXX - Cloning
      $XX XXX - Towing
```

The idea here is that these are additive (summing up your cost), but the coverage should be negative (covering all/part of your costs).  Is there a better way to do this?

```
       $XX XXX - Treatment
       $XX XXX - Revival
       $XX XXX - Cloning
       $XX XXX - Towing
      ($XX XXX) - Treatment Plan Coverage
      ($XX XXX) - Revival Plan Coverage
      ($XX XXX) - Cloning Plan Coverage
      ($XX XXX) - Towing Plan Coverage
```